packages/graph-explorer/src/modules/NodeExpand/NodeExpandContent.tsx

```diff
@@ -44,121 +62,84 @@ const NodeExpandContent = ({
   return (
-    <div className={styleWithTheme(defaultStyles(classNamePrefix))}>
-      <div className={pfx("header")}>
-        {vtConfig?.iconUrl && (
-          <div
-            className={pfx("icon")}
-            style={{
-              background: fade(vtConfig?.color, 0.2),
-              color: vtConfig?.color,
-            }}
-          >
-            <VertexIcon
-              iconUrl={vtConfig?.iconUrl}
-              iconImageType={vtConfig?.iconImageType}
-            />
-          </div>
-        )}
-        <div className={pfx("content")}>
-          <div className={pfx("title")}>
-            {displayLabels || vertex.data.type}
-          </div>
-          <div>{name}</div>
-        </div>
-      </div>
-      {vertex.data.neighborsCount === 0 && (
+    <>
+      <NeighborsList vertex={vertex} classNamePrefix={classNamePrefix} />
+      {!vertex.data.__unfetchedNeighborCount && (
         <PanelEmptyState
+          className={pfx("empty-panel-state")}
           icon={<GraphIcon />}
-          title={t("node-expand.no-connections-title")}
-          subtitle={t("node-expand.no-connections-subtitle")}
+          title={t("node-expand.no-unfetched-title")}
+          subtitle={t("node-expand.no-unfetched-subtitle")}
         />
       )}
-      {vertex.data.neighborsCount !== 0 && (
-        <>
-          <NeighborsList vertex={vertex} classNamePrefix={classNamePrefix} />
-          {!vertex.data.__unfetchedNeighborCount && (
-            <PanelEmptyState
-              className={pfx("empty-panel-state")}
-              icon={<GraphIcon />}
-              title={t("node-expand.no-unfetched-title")}
-              subtitle={t("node-expand.no-unfetched-subtitle")}
-            />
-          )}
-          {!!vertex.data.__unfetchedNeighborCount && (
-            <NodeExpandFilters
-              classNamePrefix={classNamePrefix}
-              neighborsOptions={neighborsOptions}
-              selectedType={selectedType}
-              onSelectedTypeChange={setSelectedType}
-              filters={filters}
-              onFiltersChange={setFilters}
-              limit={limit}
-              onLimitChange={setLimit}
-            />
-          )}
-          <ModuleContainerFooter>
-            <Button
-              icon={
-                isExpanding ? (
-                  <LoadingSpinner style={{ width: 24, height: 24 }} />
-                ) : (
-                  <ExpandGraphIcon />
-                )
-              }
-              variant={"filled"}
-              isDisabled={
-                isExpanding ||
-                !vertex.data.__unfetchedNeighborCount ||
-                !selectedType
-              }
-              onPress={onExpandClick}
-            >
-              Expand
-            </Button>
-          </ModuleContainerFooter>
-        </>
+      {!!vertex.data.__unfetchedNeighborCount && (
+        <NodeExpandFilters
+          classNamePrefix={classNamePrefix}
+          neighborsOptions={neighborsOptions}
```

Comment:
Fair point. I did not have a strong preference before, but after reading about it and thinking about it a bit I'm on your side. I'll start using `Boolean()` instead of `!!` for clarity.